tests/unit/modules/selenium/test_selenium_builder.py

```diff
@@ -183,6 +183,56 @@ def test_firefox_setup_generator(self):
         for idx in range(len(target_lines)):
             self.assertIn(target_lines[idx], content, msg="\n\n%s. %s" % (idx, target_lines[idx]))
 
+    def test_firefox_options_generator(self):
```

Comment:
make small unit tests for each option
1 - ignore proxy true
2 - ignore proxy false
3 - options with IE etc..